src/VecSim/algorithms/brute_force/vector_block.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <stddef.h>
+
+typedef size_t labelType;
+typedef size_t idType;
+
+// Pre declaration
+
+struct VectorBlock;
+
+struct VectorBlockMember {
+public:
```

Comment:
The `public` is not required, since in struct all fields are public by default...

Reply:
I know. readiablity and good practices >> common knowledge